packages/@okta/vuepress-site/docs/guides/quickstart/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: QuickStart
```

Comment:
Does this title also need to be updated?

Reply:
I haven't noticed that string used anywhere--but please let me know if that's wrong. I would ordinarily correct it anyways, so that it's there if it's used in the future, but the colon in the new title messes up the yaml parsing. (I tried putting the string in quotes to get around that, but building fails anyways.) So, for now, I'll leave it.